### ISSUE TITLE
Spike - Guarantee Sequential Execution of Re-parse Command

### ISSUE THREAD

**Issue by elipe17** [raft-tech/TANF-app#3065]
**Description:**
As a follow on to #3064, we would like to further explore and implement preventing the `clean_and_reparse` command from executing in parallel. The model that gets created via #3064 will have a "re-parse completed" datetime field associated with it. If it is not the case that the datetime field is set once the files associated with the latest re-parse have finished parsing that should implemented here. From there, the `clean_and_reparse` command need only grab the latest re-parse model and verify it has completed by way of the datetime field.

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] Re-parse commands are guarenteed to run sequentially
- [ ] Re-parse model "re-parse completed" datetime field has the correct datetime logic
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Implement datetime logic if not already done
- [ ] Update `clean_and_reparse` to leverage that and print information messages if it cannot run
- [ ] Add integration testing for parallel `clean_and_reparse` commands
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
- [This Celery feature ](https://docs.celeryq.dev/en/stable/userguide/calling.html#results-options)could be useful when trying to implement the correct logic for the end of re-parse datetime.

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 

- _Open Question 1_
- _Open Question 2_

**Comment by robgendron** [comment]
a "dumb" implementation for the strech goal is a task added last to the queue that polls for the completed statuses of the datafile summary and only completes once they all indicate complete.

a maybe dumbber solution is to write dfs statuses IN the aggregate model in a json field or something (thinking of it as a document db use-case). then compute based on that

**Comment by andrew-jameson** [comment]
Closed as part of #3126
